python_modules/dagster/dagster/_core/definitions/repository_definition/repository_definition.py

```diff
@@ -133,6 +142,38 @@ def __init__(
     def repository_load_data(self) -> Optional[RepositoryLoadData]:
         return self._repository_load_data
 
+    @experimental
```

Comment:
I don't think we need to make this `@experimental` as of now as it is not public